presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Add feature and session toggles for *top-N row number* optimization.
+* Add ``ngrams`` function to generate the N-grams from an array.
```

Comment:
The previous release notes actually included links to the documentation, let's consider?  Same applies to below too.

Reply:
```
* Add :func:`ngrams` function to generate the N-grams from an array.